### PR TITLE
PXB-1908: Release notes - v8.0.7

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.7.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Downloads are available from our `download site
+<https://www.percona.com/downloads/Percona-XtraBackup-LATEST/>`_ and from
+:ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+|pxb.name| enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot
+tolerate long periods of downtime. Offered free as an open source solution, it
+drives down backup costs while providing unique features for |MySQL| backups.
+
+In release |release|, |pxb.name| introduces the encryption of the system
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.7.rst]: xtrabackup does not introduce "encryption of the system tablespace", it now supports backups and prepare of the databases containing encrypted system tablespace (https://www.percona.com/doc/percona-server/5.7/management/data_at_rest_encryption.html#innodb_sys_tablespace_encrypt)

also, xtrabackup now supports encrypted mysql tablespace as well

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.7.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+tablespace. Also, this new release supports the ``lz4`` compression algorithm
+directly which improves the speed of compression. Use `--compress=lz4` to use
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.7.rst]: could be better to say that compressed backups can now use lz4 compression algorithm in addition to quicklz

**Review by gl-sergei — CHANGES_REQUESTED** [review]

**Review by gl-sergei — APPROVED** [review]

**Review by gl-sergei — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/howtos/recipes_xbk_compressed.rst [hunk 3]
```diff
@@ -43,7 +50,13 @@ You can use the following one-liner to uncompress all the files:
 
    $ for bf in `find . -iname "*\.qp"`; do qpress -d $bf $(dirname $bf) && rm $bf; done
 
-You can decompress the backup by using the ``--decompress`` option:
+If you used the ``lz4`` compression algoritm change this script to search for ``*.lz4`` files:
```
Comment (tduzan-te) [inline storage/innobase/xtrabackup/doc/source/howtos/recipes_xbk_compressed.rst:53]: `algorithm` is misspelled here

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst [hunk 4]
```diff
@@ -125,6 +136,10 @@ Options
    The :option:`--decompress` option may be used with |xbstream| to
    decompress individual qpress files.
 
+   If you used the ``lz4`` compression algoritm to compress the files
```
Comment (tduzan-te) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst:139]: `algorithm` is misspelled here

**Review by tduzan-te — COMMENTED** [review]
two misspellings, otherwise okay by me

**Pull request merged**